examples/encfs/README.md

```diff
@@ -66,22 +75,88 @@ The AAD token with permission to AKV/mHSM can be obtained with the following com
+The default logging is `warning` level. At this point, the `aci-arm-template.json` file should be filled out except for the `ccePolicy` field. 
+Set the value of the `ccePolicy` to an empty string. After installing the [Azure `confcom` CLI extension](#policy-generation), run the following command to generate the security policy and include the `--debug-mode` option so that the policy allows users to shell into the container. 
+The `--debug-mode` option is only needed for testing purposes and should not be used in production, as it allows users to shell into the containers. 
+Note this should only be used for debugging and not recommended for production systems. 
+We are using this option since this is an example run. 
+
+```
+az confcom acipolicygen -a aci-arm-template.json --debug-mode
+```
+
+This should automatically populate the [cce policy](aci-arm-template.json#L112) field of `aci-arm-template.json.` 
+If you run the tool on an ARM template with the `ccePolicy` field already populated, the tool will prompt you to overwrite the existing policy.
+
+The security policy tool outputs the sha-256 hash of the policy upon completion. 
+Copy this output and replace the [hash-digest-of-the-security-policy](importkeyconfig.json#L22) string of the `importkeyconfig.json` file.
+
+
+#### 7. Import Keys into mHSM/AKV
+
+Once the key vault resource is ready and the `importkeyconfig.json` file is completely filled out, the user can import `RSA-HSM` or `oct-HSM` keys into it using the `importkey` tool placed under `<parent_repo_dir>/tools/importkey` as discussed in the tools' [readme file](https://github.com/microsoft/confidential-sidecar-containers/tree/main/tools/importkey).
+
+To import the key into AKV/mHSM, use the following command:
+
+```
+go run tools/importkey/main.go -c importkeyconfig.json -out=true
+```
+If user configured `RSA-HSM` key type on `importkeyconfig.json`, make sure the number of characters of the salt string is even number. 
+Upon successful import completion, you should see something similar to the following: 
+
+```
+[34 71 33 117 113 25 191 84 199 236 137 166 201 103 83 20 203 233 66 236 121 110 223 2 122 99 106 20 22 212 49 224]
+https://<mhsm-name>.managedhsm.azure.net/keys/doc-sample-key-release/8659****0cdff08
+{"version":"0.2","anyOf":[{"authority":"https://sharedeus2.eus2.test.attest.azure.net","allOf":[{"claim":"x-ms-sevsnpvm-hostdata","equals":"aaa7***7cc09d"},{"claim":"x-ms-compliance-status","equals":"azure-compliant-uvm"},{"claim":"x-ms-sevsnpvm-is-debuggable","equals":"false"}]}]}
+```
+
+In this case, use the following commands to verify the key has been successfully imported: 
+
+```
+az account set --subscription "<SUBSCRIPTION>"
+az keyvault key list --hsm-name <mHSM NAME> -o table
+```
+The main.go golang script generates a private rsa key or an oct key named `keyfile.bin` based on the key type configuration on `importkeyconfig.json`.

```

Comment:
Until we update the importkey tool, it doesn't create the oct key. generatefs.sh creates the oct key. So just remove the mention of oct key for now and once we merge this, I can work to make it so that importkey creates the keys and I'll update the README accordingly.

Reply:
The import key tool actually also create the oct key as well.

Reply:
Huh, then we should just remove key generation from the generatefs.sh script